app/src/main/java/com/geeksville/mesh/database/MeshtasticDatabase.kt

```diff
@@ -27,7 +29,7 @@ import com.geeksville.mesh.database.entity.QuickChatAction
     autoMigrations = [
         AutoMigration (from = 3, to = 4),

```

Comment:
here you can simply add "_AutoMigration (from = 4, to = 5),_" instead of using the Migration object.

Reply:
I tried that originally, but it didn't seem to work. I was thinking it didn't support adding columns, but maybe I did something wrong

Reply:
Now that I think about it, I might not have had the new schema file added yet.. I'll try it again